tests/tests_pytorch/callbacks/test_throughput_monitor.py

```diff
@@ -167,23 +167,6 @@ def test_throughput_monitor_fit_gradient_accumulation(tmp_path):
     model = BoringModel()
     model.flops_per_batch = 10
 
-    trainer = Trainer(
-        devices=1,
-        logger=logger_mock,
-        callbacks=monitor,
-        limit_train_batches=5,
-        limit_val_batches=0,
-        max_epochs=2,
-        log_every_n_steps=3,
-        accumulate_grad_batches=2,
```

Comment:
Instead of removing this case, can you assert the expected state under these parameters?

Reply:
I parameterized the test with log_every_n_steps now :)